contracts/ts/constants.ts

```diff
@@ -4,3 +4,11 @@ import path from "path";
 export const abiDir = path.resolve(__dirname, "..", "artifacts");
 // The directory where the contract source files are stored.
 export const solDir = path.resolve(__dirname, "..", "contracts");
+
+/**
+ * Supported verification key modes
+ */
+export enum EMode {
```

Comment:
might be a silly question, but what does "E" stand for here?

Reply:
same question XDD

Reply:
It means enum.